bootstrap.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+# Install dependencies
+apt-get install -y imagemagick redis-server postgresql bundler libpq-dev nodejs
```

Comment:
It'd be better to install `bundler` via `gem install bundler` so that you get the latest version (and could pin the version independently of the system package's version)